recipes/mlrose/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "mlrose" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cec83253bf6da67a7fb32b2c9ae13e9dbc6cfbcaae2aa3107993e69e9788f15e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - numpy
+    - scipy
+    - scikit-learn
```

Comment:
Those are `run` requirements, not `host`.
Under `host`, for pure python packages, you should only have `python` and `pip`.